firstSegment:
  lineEnd: [0.27617406845092773, -0.2597813010215759, 0.04999997839331627]
  lineStart: [1.735251545906067, -0.22829225659370422, 0.04999997839331627]
secondSegment:
  circleCenter: [0.2726285457611084, -0.09549416601657867, 0.05000000074505806]
  circleRadius: 0.16432538910964814
segmentLengths: [1.45941722791525, 0.5162434352251548, 1.2898658281554953, 3.2655264912959003]
thirdSegment:
  lineEnd: [1.5587148666381836, 0.09618441015481949, 0.04999997839331627]
  lineStart: [0.2691398561000824, 0.06879419833421707, 0.04999997839331627]